src/general/utils/MathUtils.cs

```diff
@@ -245,4 +245,38 @@ public static double BreakOnNaN(this double value)
 
         return value;
     }
+
+    public static float BreakOnInfinity(this float value)
```

Comment:
It would be handy to modify these functions to have a string message, giving the dev a better hint at _which_ variable is infinity, but it's not necessary since you could always dig through the stack trace.

Reply:
Yeah. The exception anyway will have the stacktrace in it so people can look there if they didn't happen to be using a debugger.